src/content/reference/react/experimental_taintUniqueValue.md

```diff
@@ -45,30 +45,78 @@ Call `taintUniqueValue` with a password, token, key or hash to register it with
 import {experimental_taintUniqueValue} from 'react';
 
 experimental_taintUniqueValue(
-  'Do not pass super secret keys to the client.',
+  'Do not pass secret keys to the client.',
   process,
-  process.env.SUPER_SECRET_KEY
+  process.env.SECRET_KEY
 );
 ```
 
 [See more examples below.](#usage)
 
 #### Parameters {/*parameters*/}
 
-* `message`: The message you want to display if the object gets passed to a Client Component. It's logged as any other Error that is thrown.
+* `errMessage`: The message you want to display if the object gets passed to a Client Component. If `value` is passed to a Client Component an Error will be thrown. `errMessage` let's you define the message displayed by this Error.
```

Comment:
This is not what it's called in the code and in TypeScript definitions.

One thing here is that this is currently just passed straight to new Error but I don't think that will be the end state. We'll likely provide more context inside the error.